src/pages/AddServers/DeployAddServers.js

```diff
@@ -0,0 +1,253 @@
+// (c) Copyright 2018 SUSE LLC
+/**
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+**/
+
+import React from 'react';
+import BaseUpdateWizardPage from '../BaseUpdateWizardPage.js';
+import { LoadingMask } from '../../components/LoadingMask.js';
+import { ErrorBanner, WarningMessage } from '../../components/Messages.js';
+import { PlaybookProgress } from '../../components/PlaybookProcess.js';
+import { translate } from '../../localization/localize.js';
+import {
+  STATUS, WIPE_DISKS_PLAYBOOK, ARDANA_GEN_HOSTS_FILE_PLAYBOOK,
+  SITE_PLAYBOOK, MONASCA_DEPLOY_PLAYBOOK
+} from '../../utils/constants.js';
+import { fetchJson } from '../../utils/RestUtils.js';
+
+
+//ansible-playbook -i hosts/verb_hosts wipe_disks.yml --limit <hostname,hostname,hostname>
+//ansible-playbook -i hosts/verb_hosts ardana-gen-hosts-file.yml"
+//ansible-playbook -i hosts/verb_hosts site.yml --limit <hostname, hostname, hostname>
+//ansible-playbook -i hosts/verb_hosts monasca-deploy.yml --tags "active_ping_checks"
+
+let PLAYBOOK_POSSIBLE_STEPS = [
+  {
+    name: WIPE_DISKS_PLAYBOOK,
+    label: translate('server.deploy.progress.wipe-disks'),
+    playbooks: [WIPE_DISKS_PLAYBOOK + '.yml'],
+    payload: {limit: {}}
+  },
+  {
+    name: ARDANA_GEN_HOSTS_FILE_PLAYBOOK,
+    label: translate('server.deploy.progress.gen-hosts-file'),
+    playbooks: [ARDANA_GEN_HOSTS_FILE_PLAYBOOK + '.yml']
+  },
+  {
+    name: SITE_PLAYBOOK,
+    label: translate('server.deploy.progress.addserver.deploy'),
+    playbooks: [SITE_PLAYBOOK + '.yml'],
+    payload: {limit: {}}
+  },
+  {
+    name: MONASCA_DEPLOY_PLAYBOOK,
+    label: translate('server.deploy.progress.update-monasca'),
+    playbooks: [MONASCA_DEPLOY_PLAYBOOK + '.yml'],
+    payload: {tags: 'active_ping_checks'}
+  }
+];
+
```

Comment:
the classes in this file have no description

Reply:
don't understand what you mean